.github/workflows/ci.yaml

```
@@ -44,3 +44,39 @@ jobs:
               uses: "docker://oskarstark/phpstan-ga"
               with:
                   args: analyse
+
+    auto-merge:
+        name: "Auto-Merge"
+
+        runs-on: "ubuntu-latest"
+
+        needs:
+            - "php-cs-fixer"
+            - "composer-normalize"
+            - "phpstan"
+
+        if: >
+            github.event_name == 'pull_request' &&
+            github.event.pull_request.draft == false && (
+              github.event.action == 'opened' ||
+              github.event.action == 'reopened' ||
+              github.event.action == 'synchronize'
+            ) && (
+              github.actor == 'dependabot[bot]'
+            )
```

The indentation looks slightly off here - assuming you prefer to indent with 4 spaces.

```suggestion
            github.event_name == 'pull_request' &&
            github.event.pull_request.draft == false && (
                github.event.action == 'opened' ||
                github.event.action == 'reopened' ||
                github.event.action == 'synchronize'
            ) && (
                github.actor == 'dependabot[bot]'
            )
```